ros/src/visualization/launch/vis_node_styx.launch

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<launch>
+    <node pkg="visualization" type="vis_node.py" name="vis_node">
+    	<param name="vis_enabled" type="bool" value="True" />
+    	<param name="vis_rate" type="int" value="5" />
+    	<param name="final_wpt_scale" type="double" value="2." />
```

Comment:
The old value for this was 4.0. Is changing this intentional?

Reply:
Yea, I changed the scaling to separate between the highway track 1 and the test lot track 2.  Track 2 is much shorter and tighter so a smaller scale is more appropriate.  I also made a separate RViz drawing file for each track with the right camera zooms and settings.